RegisterPayments/function.json

```
@@ -0,0 +1,27 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "route": "register-payments",
```

```suggestion
      "route": "payments",
```

use nouns instead of verbs for resources

---

fixed in 67bc7ae